lib/linguist/languages.yml

```diff
@@ -217,6 +217,15 @@ Augeas:
   tm_scope: none
   ace_mode: text
 
+Aurora:
+  type: programming
+  color: "#62BEAD"
+  group: Python
```

Comment:
This line means that Aurora will be counted as Python in the statistics. Is that what you want?